[docs] [integrations] update integrations page on kairosdb (#11784)
yugabyte/yugabyte-db: docs/content/latest/integrations/kairosdb.md

[KairosDB](http://kairosdb.github.io/) is a Java-based time-series metrics API that leverages Cassandra as its underlying distributed database. This page shows how it can be integrated with YugabyteDB's Cassandra-compatible YCQL API.

## Prerequisites

Before you start using the KairosDB plugin, ensure that you have:

- Java version 1.8 or later.
- The latest version of [KairosDB](https://kairosdb.github.io/docs/GettingStarted.html).
- The latest version of [YugabyteDB plugin for KairosDB](https://github.com/yugabyte/kairosdb-yb-plugin/releases/) and explore the [README] for details about the plugin.

> - A YugabyteDB cluster. Refer [YugabyteDB Quick start guide](/latest/quick-start/) to install and start a local cluster.

Reviewer: ```suggestion
- A YugabyteDB cluster. Refer to [YugabyteDB Quick start guide](/latest/quick-start/) to install and start a local cluster.
```

ycqlsh>
```

> - Run the following YCQL commands, and connect to the `kairosdb` keyspace to verify it is working:

Author: I believe when we start kairosdb with the run command , it creates the database based on the configuration parameters we pass to the conf file. @Sfurti-yb should be able to confirm this.

> - Run the following YCQL commands, and connect to the `kairosdb` keyspace to verify it is working:

Reviewer 2: Yes the kairosdb keyspace is created when we use the run command

```cql
ycqlsh> describe keyspaces;
```

```output
kairosdb  system_schema  system_auth  system
```

```cql
ycqlsh> use kairosdb;
```

```cql
ycqlsh:kairosdb> describe tables;
```

```output
tag_indexed_row_keys  row_key_index  service_index  row_key_time_index
row_keys              data_points    string_index   spec
```

## Test KairosDB

- Start Postman and create a new workspace.

### Push data

> - Select the POST API request and add the following URL in the `Enter request URL` box:

Reviewer: "Select the POST API request" - can you be more specific? Do you click some button, or other UI object?

Author: Added more steps and screenshots to be more clear.

```text
http://localhost:8080/api/v1/datapoints
```

> - In the body of the request, add the following JSON, and send it.

Reviewer: ```suggestion
- In the body of the request, add the following JSON, then click **Send**.
```

```json
[
  {
      "name": "archive_file_tracked",
      "datapoints": [[1359788400000, 123], [1359788300000, 13.2], [1359788410000, 23.1]],
      "tags": {
          "host": "server1",
          "data_center": "DC1"
      },
      "ttl": 300.0
  },
  {
      "name": "archive_file_search",
      "timestamp": 1359786400000,
      "value": 321,
      "tags": {
          "host": "server2"
      }
  }
]
```

Your request should look like:

> ![kairosdb POST request](/images/develop/ecosystem-integrations/kairosdb/kairosdb1.png)

Reviewer: Possible to resize the windows of these screenshots so they doesn't show all that whitespace? They would then be easier to read, being less wide.

Author: Replaced it with the ones that showed on my Postman UI.
